tensorboard/webapp/feature_flag/views/BUILD

```diff
@@ -0,0 +1,33 @@
+load("//tensorboard/defs:defs.bzl", "tf_ng_module", "tf_sass_binary")
+
+package(default_visibility = ["//tensorboard:internal"])
+
+licenses(["notice"])
+
+tf_sass_binary(
+    name = "style",
+    src = "feature_flag_page_style.scss",
+    deps = ["//tensorboard/webapp:angular_material_sass_deps"],
+)
+
+tf_ng_module(
+    name = "feature_flag_page",
```

Comment:
Name it just "views" and therefore make it the default target. So others can refer to it as "//tensorboard/webapp/feature_flag/views"

Reply:
Done.